### PR TITLE
Add Naziel

### PR DESCRIPTION
**Pull request by Sigmatecture**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -2573,6 +2573,38 @@ export const Moves: {[k: string]: ModdedMoveData & {gen?: number}} = {
...
+			this.add('-anim', source, "High Jump Kick", target);
+		},
+		onHit(target, source) {
+			let koed: Pokemon;
+			if (this.random() < 0.55) {
+				koed = target;
+			} else {
+				koed = source;
+			}
+			koed.faint();
```
Comment (AnnikaCodes) [inline data/mods/ssb/moves.ts]:
```suggestion
			if (this.random() < 0.55) {
				target.faint();
			} else {
				source.faint();
			}
```

File: data/mods/ssb/random-teams.ts [hunk 2]
```diff
@@ -412,6 +412,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'OwO wuts dis?',
 		evs: {atk: 252, spd: 4, spe: 252}, nature: 'Adamant',
 	},
+	Naziel: {
+		species: "Kirlia", ability: 'Prankster', item: 'Eviolite', gender: ['M','F'],
```
Comment (AnnikaCodes) [inline data/mods/ssb/random-teams.ts]:
```suggestion
		species: "Kirlia", ability: 'Prankster', item: 'Eviolite', gender: '',
```

**Review by AnnikaCodes — CHANGES_REQUESTED** [review]

File: data/mods/ssb/random-teams.ts (hunk 2 above)
Comment (AnnikaCodes) [reply data/mods/ssb/random-teams.ts]: I don't think arrays of gender are supported. If it's `''`, it'll randomly pick M/F/N.

File: data/mods/ssb/moves.ts (hunk 1 above)
Comment (AnnikaCodes) [reply data/mods/ssb/moves.ts]: You could golf this even more with something like `(this.random () > 0.55 ? target : source).faint();` but I think my suggestion is more readable.

File: data/mods/ssb/random-teams.ts (hunk 2 above)
Comment (ghost) [reply data/mods/ssb/random-teams.ts]: Nah empty strings are m/f. No property is just m/f/n

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -2573,6 +2573,36 @@ export const Moves: {[k: string]: ModdedMoveData & {gen?: number}} = {
...
+		priority: 0,
+		flags: {},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, "High Jump Kick", target);
+		},
+		onHit(target, source) {
+			if (this.random() < 0.55) {
```
Comment (ghost) [inline data/mods/ssb/moves.ts]:
```suggestion
			if (this.randomChance(11, 20)) {
```

**Pull request merged**